### PR TITLE
React Calculator Milestone 2

### PR DESCRIPTION
**Pull request by macnick**
In this milestone I implemented the following: 

1. Added a `components` folder inside the `src` folder
2. Created 4 components : App, Display, Button, and ButtonPanel

Ugly page appeared after `npm start`

Please review my code.

Nick

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
```
Comment (shloch) [inline src/App.js:26]: there're 2 components you have to render here [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/appFunctionalComponent.png)]

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div>
+      <Display />
+      <ButtonPanel />
+    </div>
```
Comment (shloch) [inline src/components/App.js:10]: so you have two of these files hahaha
You need to delete one, please :) 

This should be the correct one (wrong one is the one with comment above)

File: src/components/Display.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  return <h1>{result}</h1>;
+}
+
+Display.propTypes = {
+  result: PropTypes.string,
+};
+
+Display.defaultProps = {
+  result: 0,
+};
```
Comment (shloch) [inline src/components/Display.js:14]: given that it's string, we should have 
```
  result: '0',
```

**Review by shloch — APPROVED** [review]
## Hello @macnick , 

We're good to go

accomplished

### Please there's some minor modification I expect you to do below before merging...i count on you

HAPPY CODING

File: src/components/Display.js (hunk 3 above)
Comment (macnick) [reply src/components/Display.js:14]: Fixed them both. 

Thank you for trusting me. 

Nick

**Pull request merged**